app/_data/kong_versions.yml

```diff
@@ -133,6 +133,16 @@
     cassandra: "3.x.x"
     postgres: "9.5+"
     openresty: "1.13.6.2"
+-
+  release: "1.0.x"
```

Comment:
At @thibaultcha 's suggestion, we did work to make it so we can use `1.0` for the version (or the release, or maybe both, I'm not sure) - please have a look at https://github.com/Kong/docs.konghq.com/pull/1031 and modify this PR to not use the trailing .x

Reply:
Much preferable indeed!

Reply:
Done, thanks!